ansible/playbooks/set-config.yaml

```diff
@@ -15,3 +15,27 @@
         - ../../cloudformation/apps/aem-stack-manager/
         - ../../cloudformation/apps/
         - ../../cloudformation/network/
+
+    - name: Add custom AWS Security Groups to CloudFormation template files for Publish Dispatcher ELB
+      cf_secgrps:
```

Comment:
Fine with `cf`, but please rename `secgrps` so it's consistent with the rest of security groups terminology.